diff -urN lisa-0.2.2.orig/lisarc lisa-0.2.2/lisarc
--- lisa-0.2.2.orig/lisarc	1970-01-01 00:00:00.000000000 +0000
+++ lisa-0.2.2/lisarc	2004-05-09 21:47:05.968935560 +0000
@@ -0,0 +1,16 @@
+#For more information about config see: http://lisa-home.sourceforge.net/config.html
+
+#PingAddresses = 192.168.100.0/255.255.255.0
+PingAddresses = auto
+#PingNames = someHost;someOtherHost
+#AllowedAddresses = 192.168.100.0/255.255.255.0
+AllowedAddresses = auto
+#BroadcastNetwork = 192.168.100.0/255.255.255.0
+BroadcastNetwork = auto
+FirstWait = 30                          #30 hundredth seconds
+SecondWait = -1                 	#ping a second time in hundredth seconds (-1 is disable)
+#SecondWait = 60                        #try twice, and the second time wait 0.6 seconds
+UpdatePeriod = 300                      #update after 300 secs
+DeliverUnnamedHosts = 0                 #also publish hosts without name
+MaxPingsAtOnce = 256                    #send up to 256 ICMP echo requests at once
+SearchUsingNmblookup = 1	        #when 1 also try nmblookup (require samba-client), 0 is disable
